spec/requests/foods_spec.rb

```diff
@@ -0,0 +1,25 @@
+require 'rails_helper'
+require 'capybara/rspec'
+
+RSpec.describe 'Foods', type: :feature do
+  before(:each) do
+    visit new_user_session_path
+    fill_in 'Email', with: 'lily@gmail.com'
+    fill_in 'Password', with: '123456'
+    click_button('Log in')
+  end
+
+  describe 'GET /index' do
+    it 'should visit the home page' do
+      visit root_path
+      expect(page).to have_content('Foods')
+    end
```

Comment:
- [ ] All your tests passed except this one which failed, Kindly fix this issue.
![image](https://user-images.githubusercontent.com/71874425/175695505-9af8280a-7b58-427d-ab93-41ee4bc02d42.png)

Reply:
Hi @RolandM99, in my computer this is working perfectly, can you elaborate a little about the issue? I don't know how to fix it because I cant see the bug

Reply:
Done